config/config_server.go

```diff
@@ -6,7 +6,6 @@ type ServerConfig struct {
 	Version          Version           `mapstructure:"version"`
```

Comment:
do we have default value for version ? (for backward compatibility)

Reply:
backward compatibility would be use template 2.1, will update this